src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -2322,10 +2423,14 @@ It marks an agent’s events up to a specific time as read in a customer’s cha
 |                        |                                                                                            |
 | ---------------------- | ------------------------------------------------------------------------------------------ |
 | **Method URL**         | `https://api.livechatinc.com/v3.4/agent/action/mark_events_as_seen`                        |
-| **Required scopes**    | `chats--all:ro` `chats--access:ro`                                                         |
+| **Required scopes**    | `chats--all:ro` **\*** or `chats--access:ro` **\*\***                                      |
 | **RTM API equivalent** | [`mark_events_as_seen`](/messaging/agent-chat-api/v3.4/rtm-reference/#mark-events-as-seen) |
 | **Webhook**            | [`events_marked_as_seen`](/management/webhooks/v3.4/#events_marked_as_seen)                |
 
+**\*)** `chats--all:ro` - to mark events as seen in a chat taking place in any group.
+
+**\*\*)** `chats--access:ro` - to mark events as seen in a chat taking place in any group.
```

Comment:
@bkubiak there were two descriptions to this method, so please confirm that I have used the correct one :)

Reply:
I just explained here that we have a minor bug in the method, but of course, this is internal info; let's describe it like we want this method to work (so everything before `in fact...`)